test/release/examples/primers/distributions.suppressif

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Did you give this one a try with cpu-as-device mode? We run this nightly in that mode. It'd be good to make sure.

Reply:
Yup! Passes with no issue

Reply:
Great, thanks!